verto/processors/GenericContainerBlockProcessor.py

```diff
@@ -113,3 +116,6 @@ def run(self, parent, blocks):
         parser = HtmlParser()
         parser.feed(html_string).close()
         parent.append(parser.get_root())
+
+    def custom_parsing(self, content_blocks, argument_values):
```

Comment:
Missing docstring, which should include information on how subclass can override it